application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -11,12 +11,11 @@
 import org.scilab.forge.jlatexmath.TeXFormula;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.CommandVisibility;
 import org.togetherjava.tjbot.commands.SlashCommandAdapter;
-import org.togetherjava.tjbot.commands.SlashCommandVisibility;
 
 import javax.imageio.ImageIO;
-import java.awt.Color;
-import java.awt.Image;
+import java.awt.*;
```

Comment:
NIT: id recommend u add `java.awt.*` to the forbidden star-imports in ur intellij settings. otherwise its easy to accidentally import `java.awt.List`